YACS-RCOS/yacs.n
# Make `npm run start` OS agnostic

**Issue**

closes #174

**Test Procedure**

Show procedure to test functionality with a clear procedure

> 1. Use any Windows OS
> 2. Do `npm run start` in project root (and have dependencies installed)
> 3. Backend & frontend should be successfully running

**Additional Info**

I'm not sure if we need `package-lock.json` for the project root, but I added it anyway. 🤷‍♂️
